c7n/resources/securityhub.py

```diff
@@ -673,3 +674,15 @@ def register_resource(klass, registry, resource_class):
 
 AWS.resources.subscribe(OtherResourcePostFinding.register_resource)
 AWS.resources.subscribe(SecurityHubFindingFilter.register_resources)
+
+
+
+@resources.register("aws.securityhub-finding")
+class SecurityhubFinding(query.QueryResourceManager):
+    class resource_type(query.TypeInfo):
+        service = "securityhub"
+        enum_spec = ('get_findings', 'Findings', None)
+        arn = False
+        id = "Id"
+        name = "ProductName"
```

Comment:
We probably want to have some default server-side query options on this one - in some test runs I was seeing archived/resolved/passed findings come back here and the policy took a good ~45 seconds to run.